product_docs/docs/pgd/5/security/access.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Access
```

Comment:
Maybe "Access Control"? As a standalone word, this title provides very little context.